assets/swg-button.css

```diff
@@ -61,7 +61,7 @@
5NDQxNiwxNS4xMzM1NTU5IDI0LjE5Mjg5MzQsMTYuNjM2MDYwMSAyNiwxNi42MzYwNjAxIEMyNy44MDcxMDY2LDE2LjYzNjA2MDEgMjkuMzcwNTU4NCwxNS4xNDM1NzI2IDI5LjM3MDU1ODQsMTMgWiIgaWQ9IlNoYXBlIj48L3BhdGg+CiAgICAgICAgICAgIDxwYXRoIGQ9Ik00NSwxMyBDNDUsMTYuNDU1NzU5NiA0Mi4zMDk2NDQ3LDE5IDM5LDE5IEMzNS43MDA1MDc2LDE5IDMzLDE2LjQ1NTc1OTYgMzMsMTMgQzMzLDkuNTI0MjA3MDEgMzUuNjkwMzU1Myw3IDM5LDcgQzQyLjMwOTY0NDcsNyA0NSw5LjUyNDIwNzAxIDQ1LDEzIFogTTQyLjM4MDcxMDcsMTMgQzQyLjM4MDcxMDcsMTAuODM2Mzk0IDQwLjgxNzI1ODksOS4zNjM5Mzk5IDM5LjAxMDE1MjMsOS4zNjM5Mzk5IEMzNy4yMDMwNDU3LDkuMzYzOTM5OSAzNS42Mzk1OTM5LDEwLjg0NjQxMDcgMzUuNjM5NTkzOSwxMyBDMzUuNjM5NTkzOSwxNS4xMzM1NTU5IDM3LjIwMzA0NTcsMTYuNjM2MDYwMSAzOS4wMTAxNTIzLDE2LjYzNjA2MDEgQzQwLjgxNzI1ODksMTYuNjQ2MDc2OCA0Mi4zODA3MTA3LDE1LjE0MzU3MjYgNDIuMzgwNzEwNywxMyBaIiBpZD0iU2hhcGUiPjwvcGF0aD4KICAgICAgICAgICAgPHBhdGggZD0iTTU3LDcuMzYyMTE4NiBMNTcsMTcuODkyOTE4OCBDNTcsMjIuMjI4NTU1IDU0LjUwMTMwNTUsMjQgNTEuNTQzMDgwOSwyNCBDNDguNzU3MTgwMiwyNCA0Ny4wOTEzODM4LDIyLjA5MTUzNzEgNDYuNDU5NTMsMjAuNTM1NDA1OSBMNDguNjUxODcxMiwxOS42MDU2NDE5IEM0OS4wNDQzODY0LDIwLjU2NDc2NjggNTAuMDAxNzQwNiwyMS42OTAyNzA2IDUxLjU0MzA4MDksMjEuNjkwMjcwNiBDNTMuNDM4NjQyMywyMS42OTAyNzA2IDU0LjYwNjYxNDQsMjAuNDg2NDcwOSA1NC42MDY2MTQ0LDE4LjI0NTI1MDQgTDU0LjYwNjYxNDQsMTcuNDAzNTY5NCBMNTQuNTIwNDUyNiwxNy40MDM1Njk0IEM1My45NTU2MTM2LDE4LjExODAxOTYgNTIuODY0MjI5OCwxOC43MzQ1OTk5IDUxLjQ5NTIxMzIsMTguNzM0NTk5OSBDNDguNjIzMTUwNiwxOC43MzQ1OTk5IDQ2LDE2LjE4MDE5NTcgNDYsMTIuODkxNzY3NCBDNDYsOS41ODM3NjUxMSA0OC42MzI3MjQxLDcgNTEuNDk1MjEzMiw3IEM1Mi44NjQyMjk4LDcgNTMuOTU1NjEzNiw3LjYyNjM2NzMgNTQuNTIwNDUyNiw4LjMxMTQ1NjUzIEw1NC42MDY2MTQ0LDguMzExNDU2NTMgTDU0LjYwNjYxNDQsNy4zNjIxMTg2IEw1Nyw3LjM2MjExODYgWiBNNTQuNzc4OTM4MiwxMi45MDE1NTQ0IEM1NC43Nzg5MzgyLDEwLjgzNjQ5OTcgNTMuNDI5MDY4OCw5LjMxOTUxNjQxIDUxLjcxNTQwNDcsOS4zMTk1MTY0MSBDNDkuOTczMDIsOS4zMTk1MTY0MSA0OC41MTc4NDE2LDEwLjgyNjcxMjcgNDguNTE3ODQxNiwxMi45MDE1NTQ0IEM0OC41MTc4NDE2LDE0Ljk0NzAzNTEgNDkuOTczMDIsMTYuNDM0NjU3NSA1MS43MTU0MDQ3LDE2LjQzNDY1NzUgQzUzLjQyOTA2ODgsMTYuNDM0NjU3NSA1NC43Nzg5MzgyLDE0Ljk0NzAzNTEgNTQuNzc4OTM4MiwxMi45MDE1NTQ0IFoiIGlkPSJTaGFwZSI+PC9wYXRoPgogICAgICAgICAgICA8cG9seWdvbiBpZD0iU2hhcGUiIHBvaW50cz0iNjMgMSA2MyAxOSA2MCAxOSA2MCAxIj48L3BvbHlnb24+CiAgICAgICAgICAgIDxwYXRoIGQ9Ik03Mi44NDQxMjAzLDE0Ljk3MzI4ODggTDc0Ljg4OTY5OTIsMTYuMzM1NTU5MyBDNzQuMjI3ODk0MywxNy4zMTcxOTUzIDcyLjYzMzU0NiwxOSA2OS44NzYwMjU1LDE5IEM2Ni40NTY3MDAxLDE5IDY0LDE2LjM2NTYwOTMgNjQsMTMgQzY0LDkuNDM0MDU2NzYgNjYuNDg2NzgyMSw3IDY5LjU4NTIzMjUsNyBDNzIuNzAzNzM3NSw3IDc0LjIyNzg5NDMsOS40NzQxMjM1NCA3NC43MjkyNjE2LDEwLjgxNjM2MDYgTDc1LDExLjQ5NzQ5NTggTDY2Ljk2ODA5NDgsMTQuODIzMDM4NCBDNjcuNTc5NzYzLDE2LjAyNTA0MTcgNjguNTQyMzg4MywxNi42NDYwNzY4IDY5Ljg4NjA1MjksMTYuNjQ2MDc2OCBDNzEuMjI5NzE3NCwxNi42NDYwNzY4IDcyLjE2MjI2MDcsMTUuOTc0OTU4MyA3Mi44NDQxMjAzLDE0Ljk3MzI4ODggWiBNNjYuNTM2OTE4OSwxMi44MDk2ODI4IEw3MS45MTE1NzcsMTAuNTg1OTc2NiBDNzEuNjEwNzU2Niw5LjgzNDcyNDU0IDcwLjcyODM1LDkuMzEzODU2NDMgNjkuNjc1NDc4Niw5LjMxMzg1NjQzIEM2OC4zMzE4MTQsOS4zMTM4NTY0MyA2Ni40NjY3Mjc0LDEwLjQ5NTgyNjQgNjYuNTM2OTE4OSwxMi44MDk2ODI4IFoiIGlkPSJTaGFwZSI+PC9wYXRoPgogICAgICAgIDwvZz4KICAgIDwvZz4KPC9zdmc+');
```

Comment:
Is this change necessary? It looks like it significantly adds to the size?

Reply:
Updated the encoding.